prow/cmd/sinker/main.go

```diff
@@ -360,45 +357,6 @@ func (c *controller) clean() {
 		}
 	}
 
-	// Now clean up old pods.
```

Comment:
I think we can completely get rid of the pod clients in sinker now (l133-142 & remove the `podClient` property from the `controller` struct)